app/uk/gov/hmrc/tariffclassificationfrontend/models/CommodityCode.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.models
+
+import java.time.{Clock, Instant}
+
+case class CommodityCode
+(
+  code: String,
+  expiry: Option[Instant] = None
+) {
+  def isExpired(implicit clock: Clock): Boolean = expiry.exists(_.isBefore(clock.instant()))
+  def isLive(implicit clock: Clock): Boolean = !isExpired
```

Comment:
just something curious:  if expiry does not exist the assumption is that the code is forever alive?

Reply:
Correct :)